cmd/agent/agent_create.go

```diff
@@ -75,25 +87,30 @@ func NewCmdCreateAgent(configuration *config.Configuration) *cobra.Command {
 		Long:    agentLong,
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			if err := o.Complete(configuration); err != nil {
+			if err := options.WithConfiguration(configuration); err != nil {
 				return err
 			}
 
-			if err := o.Validate(); err != nil {
+			if err := options.Validate(); err != nil {
 				return err
 			}
 
-			if err := o.Run(); err != nil {
+			if err := options.Run(); err != nil {
 				return err
 			}
 			return nil
 		},
 		SilenceUsage: true,
 	}
+
+	cmd.Flags().StringVarP(&options.ContextName, "context-name", "", "", "specify the name of the kubernetes context to select from your kube config")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&options.ContextName, "context-name", "", "", "specify the name of the Kubernetes context to select from your kubeconfig")
```

The Kubernetes documentation calls it a "kubeconfig" (one word): https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/

Reply:
This one doesn't say "Skips prompt" but I would have thought that it does?

Reply:
Yes, yes it does :-D